xpdacq/md_schema_v1.json

```diff
@@ -0,0 +1,153 @@
+{
+  "type": "object",
+  "properties":{
+    "bt_piLast":{
```

Comment:
yup, you got the point. we had nested ``dict`` at first but it turns out searching nesting ``dict`` was a huge pain.

We can discuss about it again since now db has filter method